src/PKSim.Core/Commands/SetExpressionLocalizationCommand.cs

```diff
@@ -110,8 +110,8 @@ private IEnumerable<IOSPSuiteCommand> updateTissueExpressionParameters(IExecutio
       {
          var command = new PKSimMacroCommand();
          //We need to iterate over all parameters defined in the protein and update the values as expected
-         var allInterstitialFractionParameters = _simulationSubject.Individual.GetAllChildren<IParameter>(x =>
-            x.IsNamed(CoreConstants.Parameters.FRACTION_EXPRESSED_INTERSTITIAL) && x.ParentContainer.IsNamed(_protein.Name));
+         var allIntracellularFractionParameters = _simulationSubject.Individual.GetAllChildren<IParameter>(x =>
+            x.IsNamed(CoreConstants.Parameters.FRACTION_EXPRESSED_INTRACELLULAR) && x.ParentContainer.IsNamed(_protein.Name));
```

Comment:
@actually it makes more sense to always have interstitial be the formula (1-fcell). That way, it is symetrical to apical/basolateral and bc membrane stuff